docs/package.json

```diff
@@ -9,12 +9,20 @@
     "help": "docusaurus --help",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "write-i18n": "npm run write-heading-ids && npm run write-translations",
     "format": "prettier --check \"**/*.{js,jsx,json,md,scss}\"",
     "format:fix": "prettier --write \"**/*.{js,jsx,json,md,scss}\"",
     "lint": "set -ex; eslint .",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "crowdin": "crowdin",
+    "crowdin:download": "npm run write-i18n && npm run crowdin download && npm run format:fix",
+    "crowdin:sync": "npm run write-i18n && npm run crowdin upload && npm run crowdin download && npm run format:fix",
+    "prebuild": "npm run crowdin:download"
```

Comment:
I would like to be able to build the english docs without downloading other translations

Reply:
Removed it.